lib/service/storage/elasticsearch.ts

```diff
@@ -2644,7 +2857,17 @@ class ElasticSearch extends Service {
    *
    * @returns {Promise.<{ documents, errors }>
    */
-  async mDelete(index, collection, ids, { refresh, timeout } = {}) {
+  async mDelete(
+    index: string,
+    collection: string,
+    ids: string[],
+    {
+      refresh,
+    }: {
+      refresh?: boolean | "wait_for";
+      timeout?: number;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
    }: Pick<KRequestParams, "refresh" | "timeout"> = {},
```

With new `KRequestParams` type we can compose type like it, to DRY